### DISCUSSION TITLE
Output Just EOL OS

### DISCUSSION THREAD

**Discussion by adambarnett52** [aquasecurity/trivy#7144] [Q&A]
### Question

Hi, 

I am looking to just output the report if the docker image is running an EOL OS

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Operating System

Ubuntu 

### Version

```bash
Version: 0.53.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-07-10 06:11:53.827384478 +0000 UTC
  NextUpdate: 2024-07-10 12:11:53.827384067 +0000 UTC
  DownloadedAt: 2024-07-10 09:45:50.566343866 +0000 UTC
```

**Comment by knqyf263** [comment]
It will be included in the JSON report if it reaches EOL.
https://github.com/aquasecurity/trivy/blob/d1f89672d92d04504808f9fa02845282c5e81b26/pkg/fanal/types/artifact.go#L12

**Comment by adambarnett52** [comment]
Thanks a little bit of `jq` let me parse it
